src/mmrelay/meshtastic/packet_routing.py

```
@@ -0,0 +1,179 @@
+        return frozenset(), frozenset()
+    routing_section = meshtastic_section.get(CONFIG_KEY_PACKET_ROUTING, {})
+    if not isinstance(routing_section, dict):
+        return frozenset(), frozenset()
+    chat_portnums = _resolve_portnum_set(routing_section.get(CONFIG_KEY_CHAT_PORTNUMS))
+    disabled_portnums = _resolve_portnum_set(
+        routing_section.get(CONFIG_KEY_DISABLED_PORTNUMS)
+    )
+    return chat_portnums, disabled_portnums
+
+
+def classify_packet(portnum: Any, config: dict[str, Any] | None) -> str:
```

Ruff (0.15.9)

[warning] Too many return statements (7 > 6)

(PLR0911)